src/com/android/messaging/datamodel/ParticipantRefresh.java

```diff
@@ -390,8 +390,8 @@ private static void refreshSelfParticipantList() {
             // For subscriptions already in the database, refresh ParticipantColumns.SIM_SLOT_ID.
             for (final Integer subId : activeSubscriptionIdToRecordMap.keySet()) {
                 final SubscriptionInfo record = activeSubscriptionIdToRecordMap.get(subId);
-                final String displayName =
-                        DatabaseUtils.sqlEscapeString(record.getDisplayName().toString());
```

Comment:
You can use `Objects.toString(record.getDisplayName(), "")` instead and just keep it on one line.

https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#toString-java.lang.Object-java.lang.String-